ci-operator/step-registry/optional-operators/README.md

```diff
@@ -0,0 +1,36 @@
+# Troubleshooting olm deployment failures
+
+This document should help you to analyze where to look in case that olm deployment has failed. It also describes workflows and which one should be your main concern. Logs location is also mentioned as one of the main subjects.
+* [Workflows](#Workflows):
+* [Logs and how to get them](#Logs-and-how-to-get-to-them):
+
+## Workflows:
+You can find all the workflows that are being executed in following yaml files:
+https://github.com/openshift/release/tree/master/ci-operator/config/redhat-openshift-ecosystem/cvp
+
+I will use `4.7` ocp version in this example:
```

Comment:
First person, try something like:

This example uses the `4.7` ocp version workflow